common/src/View/EntityAttributeGridTable.cpp

```diff
@@ -114,7 +114,7 @@ namespace TrenchBroom {
         }
         
         bool EntityAttributeGridTable::RowManager::isDefaultRow(const size_t rowIndex) const {
-            assert(rowIndex < totalRowCount());
+            ensure(rowIndex < totalRowCount(), "row index must be < totalRowCount()");
```

Comment:
Please use "row index out of bounds" and adapt all the other error messages for index bounds checks.